docs/en/install-upgrade/release-notes/release-notes-logstash-rc1.asciidoc

```diff
@@ -1,3 +1,131 @@
 = {ls} version 9.0.0-rc1
 
-coming::[9.0.0-rc1]
\ No newline at end of file
+coming::[9.0.0-rc1]
+
+== Breaking changes
+
+[[pipeline-buffer-type]]
+=== Pipeline buffer type changed to 'heap'
+
+Set `pipeline.buffer.type` to `direct` to continue using direct memory with {agent}, {beats}, TCP, and HTTP input plugins https://github.com/elastic/logstash/pull/16500[#16500]
+
+[[removed-params]]
+=== Removed deprecated params
+
+`http.*` params replaced by `api.*` are removed https://github.com/elastic/logstash/pull/16552[#16552]
+
+`event_api.tags.illegal` no longer exists. 
+Instead, {ls} generates `_tagsparsefailure` in the event `tags` and the illegal value is written to the `_tags` field https://github.com/elastic/logstash/pull/16461[#16461]
+
+[[removed-modules]]
+=== Removed {ls} modules
+
+Removed netflow, fb_apache and azure https://github.com/elastic/logstash/pull/16514[#16514]
+
+Removed Arcsight module and the modules framework https://github.com/elastic/logstash/pull/16794[#16794]
+
+Removed ingest-converter https://github.com/elastic/logstash/pull/16453[#16453]
+
+[[jdk-11-support-drop]]
+=== JDK 11 support drop
+
+Starting with {ls} 9.0, we no longer support Java 11 https://github.com/elastic/logstash/pull/16443[#16443]
+
+[[docker-base-image-change]]
+=== Docker base image change
+
+{ls} on Docker, Ubuntu base image changed to UBI https://github.com/elastic/logstash/pull/16599[#16599]
+
+[[allow-superuser]]
+=== {ls} does not run with superuser by default
+
+Starting with 9.0, set `allow_superuser` to `true` if you prefer running {ls} as a superuser https://github.com/elastic/logstash/pull/16558[#16558]
+
+[[allow-legacy-monitoring]]
+=== {ls} introduces new setting to allow legacy internal monitoring
+
+Starting from 9.0, set `xpack.monitoring.allow_legacy_collection` to `true` to explicitly permit legacy monitoring https://github.com/elastic/logstash/pull/16586[#16586]
+
+[[avoid-collision-on-json-fileds]]
+=== {ls} avoids JSON log lines collision
+
+`log.format.json.fix_duplicate_message_fields` config is enabled by default to avoid collision on `message` field in log lines when `log.format` is JSON https://github.com/elastic/logstash/pull/16578[#16578]
+
+
+[[logstash-900-rc1-features-enhancements]]
+== Features and enhancements 
+
+* Use UBI9 as base image https://github.com/elastic/logstash/pull/17174[#17174]
+* Improve plugins remove command to support multiple plugins https://github.com/elastic/logstash/pull/17030[#17030]
+* Allow concurrent Batch deserialization https://github.com/elastic/logstash/pull/17050[#17050]
+* Upgrade filter-xml and its nokogori dependency https://github.com/elastic/logstash/pull/17130[#17130]
+* Update rack dependency to 3.1.11 https://github.com/elastic/logstash/pull/17230[#17230]
+* Upgrade elasticsearch-ruby client to v8 https://github.com/elastic/logstash/pull/17161[#17161]
```

Comment:
Sorry @karenzone for noises. I would like to remove these items to better align with a consolidation approach.
```suggestion
```